src/sentry/api/endpoints/project_triage_stats.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from datetime import timedelta
+from django.utils import timezone
+
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.models import Group, GroupStatus, Activity
+from sentry.utils.dates import to_timestamp
+
+
+class TriageStatus(GroupStatus):
+    DNE = -1
+    ASSIGNED = 6
+
+
+class ProjectTriageStatsEndpoint(ProjectEndpoint):
+
+    def filter_activity_set(self, set, cutoff):
+        return [ai for ai in set if ai.datetime < cutoff]
+
+    def process_activity_set(self, group, dates):
+
+        activity_set = Activity.objects.filter(
```

Comment:
Add in a `.using('default_analytics')` to point this against the replica just for staging.

So `Activity.objects.using('default_analytics').filter(...)`